src/services/interview.ts

```diff
@@ -1,39 +1,58 @@
 import { DetailedResponse, Interview, InterviewAvailability } from '../types';
 import api from './api';
 
+function deserializeInterview(data: any): Interview {
```

Comment:
You might want to use unknown or create a type instead of any in case there are issues with the data. Also why is this necessary, looks like you're just renaming fields.

Reply:
I will add a type. I essentially am just renaming types, but i didn't want to rewrite the code to do so over and over for each api call